crawl-ref/source/dat/descript/backgrounds.txt

```diff
@@ -173,6 +173,12 @@ Monk
 Monks start with a simple weapon of their choice, and gain additional piety
 with the first god they worship.
 %%%%
+Neckbeard
```

Comment:
Needs to be just two lines (at 80char width)

Reply:
Fixed